dbt/sagerx/models/staging/rxnorm/_rxnorm__models.yml

```diff
@@ -169,6 +169,8 @@ models:
       - name: name
       - name: tty
       - name: rxaui
+        tests:
```

Comment:
INFO
In dbt v1.8, what was previously known as "tests" are now called "data tests" with the addition of [unit tests](https://docs.getdbt.com/docs/build/unit-tests). The YAML key tests: is still supported as an alias for data tests but will be deprecated in the future in favor of data_tests:. Refer to [New data_tests: syntax](https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax) for more information.